angr/storage/memory_mixins/address_concretization_mixin.py

```diff
@@ -270,13 +270,13 @@ def load(self, addr, size=None, *, condition=None, **kwargs):
             )
 
         if self.state.solver.symbolic(addr) and options.AVOID_MULTIVALUED_READS in self.state.options:
-            return self._default_value(None, size, name="symbolic_read_unconstrained", **kwargs)
+            return self._default_value(None, size, name=f"symbolic_read_unconstrained_({addr})", **kwargs)
```

Comment:
Minor nitpick, can you remove the parenthesis here and below? I had to do a double take because I thought it was a `{{` at first.

Reply:
Resolved